dymos/test/test_run_problem.py

```diff
@@ -16,13 +17,14 @@
 from openmdao.utils.general_utils import set_pyoptsparse_opt
 _, optimizer = set_pyoptsparse_opt('IPOPT', fallback=True)
 
+om_version = tuple([int(s) for s in openmdao.__version__.split('-')[0].split('.')])
```

Comment:
I wonder if it might be worthwhile to compute this tuple in openmdao.api to provide a convenient way to check openmdao version.